### ISSUE TITLE
tracing: sysview: pm_device_runtime_put trace function generates compile error

### ISSUE THREAD

**Issue by mbung** [zephyrproject-rtos/zephyr#73159]
**Describe the bug**
When enabling tracing via segger systemview a compile error occurs for the macro `sys_port_trace_pm_device_runtime_put_async_enter(dev, delay)` due to too many parameters for
`SEGGER_SYSVIEW_RecordU32()`. Two given, but only one expected.
- Zephyr v3.6.0
- modules/debug/segger at commit `9d0191285956cef43daf411edc2f1a7788346def ` (manifest revision)

**To Reproduce**
1. Enable tracing via SystemView
2. Enable power management
3. build project
4. Compile error in tracing_sysview.h thrown

**Expected behavior**
Application builds without compile error.

**Impact**
Can't enable tracing in apps where pm is used.

**Logs and console output**
```shell
/workdir/zephyr/subsys/tracing/sysview/./tracing_sysview.h:653:9: error: too many arguments to function 'SEGGER_SYSVIEW_RecordU32'
  653 |         SEGGER_SYSVIEW_RecordU32(TID_PM_DEVICE_RUNTIME_PUT_ASYNC,              \
      |         ^~~~~~~~~~~~~~~~~~~~~~~~
```

**Additional context**
To solve the issue `SEGGER_SYSVIEW_RecordU32` needs to be changed to `SEGGER_SYSVIEW_RecordU32x2`.

**Comment by nashif** [comment]
@mbung  do you want to send a PR fixing this?

**Comment by hatpool** [comment]
I try although it's my first time contributing to zephyr. I contribute from this private account to avoid discussions with my employer.

Actually I've issues testing my fix on the current main branch. I tried to build with `samples/subsys/pm/device_pm` for the board `nrf52840dk/nrf52840`. But the build refuses to enable PM:

```shell
Loading Zephyr default modules (Zephyr base).
-- Application: /workdir/zephyr/samples/subsys/pm/device_pm
-- CMake version: 3.22.1
-- Found Python3: /usr/bin/python3 (found suitable version "3.10.12", minimum required is "3.8") found components: Interpreter 
-- Cache files will be written to: /home/user/.cache/zephyr
-- Zephyr version: 3.6.99 (/workdir/zephyr)
-- Found west (found suitable version "1.2.0", minimum required is "0.14.0")
-- Board: nrf52840dk, qualifiers: nrf52840
-- Found host-tools: zephyr 0.16.5 (/opt/toolchains/zephyr-sdk-0.16.5-1)
-- Found toolchain: zephyr 0.16.5 (/opt/toolchains/zephyr-sdk-0.16.5-1)
-- Found Dtc: /opt/toolchains/zephyr-sdk-0.16.5-1/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /workdir/zephyr/boards/nordic/nrf52840dk/nrf52840dk_nrf52840.dts
-- Generated zephyr.dts: /workdir/zephyr/samples/subsys/pm/device_pm/build/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /workdir/zephyr/samples/subsys/pm/device_pm/build/zephyr/include/generated/zephyr/devicetree_generated.h
-- Including generated dts.cmake file: /workdir/zephyr/samples/subsys/pm/device_pm/build/zephyr/dts.cmake

warning: PM (defined at soc/silabs/silabs_s2/efr32bg22/Kconfig.defconfig:12,
soc/silabs/silabs_s2/efr32bg27/Kconfig.defconfig:12,
soc/silabs/silabs_s2/efr32mg24/Kconfig.defconfig:12, subsys/pm/Kconfig:13) was assigned the value
'y' but got the value 'n'. Check these unsatisfied dependencies: ((SOC_SERIES_EFR32BG22 &&
SOC_VENDOR_SILABS) || (SOC_SERIES_EFR32BG27 && SOC_VENDOR_SILABS) || (SOC_SERIES_EFR32MG24 &&
SOC_VENDOR_SILABS) || (SYS_CLOCK_EXISTS && HAS_PM)) (=n). See
http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_PM and/or look up PM in the
menuconfig/guiconfig interface. The Application Development Primer, Setting Configuration Values,
and Kconfig - Tips and Best Practices sections of the manual might be helpful too.
```

I'm not yet familiar with the new hardware model since this is not yet released. I tried to build with `west build -b nrf52840dk/nrf52840`. Any suggestions welcome to fix the unintended use of the silabs board.

I'm working on the latest zephyr devcontainer.

**Comment by hatpool** [comment]
Never mind, [Wojciech Slenska] fixed it already 3 months ago. Sorry, didn't check the main branch. Can be closed.

**Comment by nashif** [comment]
fixed in d079701885c104cadf1667dfd1e940ed4f7e8fbb
